pkg/odo/cli/storage/delete.go

```diff
@@ -99,6 +100,7 @@ func NewCmdStorageDelete(name, fullName string) *cobra.Command {
 
 	storageDeleteCmd.Flags().BoolVarP(&o.storageForceDeleteFlag, "force", "f", false, "Delete storage without prompting")
 	completion.RegisterCommandHandler(storageDeleteCmd, completion.StorageDeleteCompletionHandler)
+	storageDeleteCmd.PersistentFlags().BoolVarP(&o.wait, "wait", "w", false, "Wait until the storage is gone")
```

Comment:
IMHO, it's worth mentioning that `storage is gone from OpenShift`. because as far as component created by `odo` is concerned, we still need to do `odo push` after `odo storage delete` to make sure that mount point goes away. Correct me if I'm wrong.